jobs/Software_Engineer_Molex.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Software Engineer
+company: Molex, LLC
+location: Maumelle, AR
+contract: permanent
+contact:
+  name: Annette Wise
+  email: Annette.Wise@molex.com
+  phone: +1-501-851-4850 x8516555
+created: !!timestamp '2017-01-31'
+tags:
+  - us
+  - python
+  - numpy
+  - wxPython
+  - PyQt
+  - architecture
+  - svn
+  - git
+  - scientific
+---
+
+# Position Overview
+
+Molex is seeking a Software Engineer enthusiast to develop, integrate, and deploy software to facilitate the design, manufacture, and
+testing of high-speed electrical interconnects across multiple manufacturing regions throughout the world.  In this role you will
+coordinate software development activities with various teams, including product development, manufacturing, marketing, quality, etc..
+
+# Essential Functions
+
+* Work closely with electrical and mechanical engineers to write software specifications to meet internal and external customer requirements and design and develop software solutions for both internal and external customers.
+* Evaluate new software solution opportunities for existing challenges and ensure compliance and safety throughout software life cycle.
+* Evaluate software reliability and performance using code reviews, design reviews, and unit testing. Analyze test results, prepare reports, and make required software modifications. Write API and usage documentation, and publish it internally.
+* Develop statements of work before all significant developments to ensure customer needs are met while reducing scope creep. Develop project timelines and milestone dates and use risk mitigation tools like code and design reviews to ensure product success. 
+
+# Required Qualifications
+
+* Bachelor's degree in Systems or Computer Engineering or Computer Science
+* 3+ years' software engineering experience
```

Comment:
Generally, we advise against requiring years of experience. People learn at different rates and this might exclude a perfectly qualified quick-learner.